src/clr-angular/forms/styles/_mixins.forms.scss

```diff
@@ -136,3 +136,24 @@
     outline: 0;
   }
 }
+
+@mixin range-style($height, $color) {
+  -webkit-appearance: none;
+  left: 0;
+  height: $height;
+  background-color: $color;
```

Comment:
We will need to use custom properties here. Check out how `@include css-var()` is used in other SASS files. `background-color`, `background`, and `border-radius` should all be allowed to use custom properties.

Reply:
...